docs/client/components/pages/Emoji/index.js

```diff
@@ -136,7 +136,15 @@ export default class App extends Component {
           </div>
           <div className={styles.param}>
             <span className={styles.paramName}>imagePath</span>
-            <span>The Emojis are displayed using SVGs. The full path is constructed of multiple parts like this: {'`${imagePath}${unicode}.svg${cacheBustParam}`'}. The default imagePath is: '//cdn.jsdelivr.net/emojione/assets/svg/', but can be overwritten with this config.</span>
+            <span>The Emojis are displayed using SVGs. The full path is constructed of multiple parts like this: {'`${imagePath}${unicode}.${imageType}${cacheBustParam}`'}. The default imagePath is: '//cdn.jsdelivr.net/emojione/assets/svg/', but can be overwritten with this config.</span>
+          </div>
+          <div className={styles.param}>
+            <span className={styles.paramName}>imageType</span>
+            <span>The default imageType is: '.svg', but can be overwritten with this config.</span>
```

Comment:
Can you change it from `.svg` to `svg`? Is it correct that EmojiOne only provides `png` as an alternative option. If yes, can you mention it here?